src/main/java/com/nguyenquyhy/discordbridge/utils/ColorUtil.java

```diff
@@ -65,4 +68,11 @@ public static TextColor getColor(Color color) {
         }
         return result;
     }
+    public static String removeColor(String text){
```

Comment:
I feel like this is a band-aid fix for a bigger problem... We should probably override `sendMessage` to accept Text objects. Ie
```java
    public static void sendMessage(Channel channel, Text content){
        sendMessage(channel, content.toPlain());
    }
```
Would allow us to use the method sponge provides for stripping formating insead of adding a new one.